app/controllers/photos_controller.rb

```diff
@@ -21,6 +21,8 @@ def new
     @id = params[:id]
 
     @photo = Photo.new
+    item_class = Growstuff::Constants::PhotoModels.get_item(params[:type])
```

Comment:
this is good -- do you think it can go into a method and the code be called from 2 places, instead of it repeating?

Reply:
Definitely! I will refactor it soon.